prac_10/testing.py

```diff
@@ -0,0 +1,90 @@
+"""
+CP1404/CP5632 Practical
+Testing demo using assert and doctest
+"""
+
+import doctest
+from prac_06.car import Car
+
+
+def repeat_string(input_string, number_of_times_repeated):
+    """Repeat input_string a given amount of times with spaces in between."""
+    repeated_strings = []
+    for i in range(number_of_times_repeated):
```

Comment:
Can this for loop be simplified by using string multiplier "*" so that the string is multiplied by the number_of_times_repeated. eg. [input_string] * number_of_times_repeated